tests/yul/c2c_test.py

```diff
@@ -111,6 +133,7 @@ async def test_starknet():
         caller_address=0,
     )
 
+    print(mint_res)
```

Comment:
Need to remove prints in this file

Reply:
We can just leave them, pytest doesn't print them anyway

Reply:
Why have them then?) It's definitely not something serious, but they just don't need to be there

Reply:
When you're working on them locally they can make things faster to see, you can just pass a flag to see the output